helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -144,13 +144,18 @@ distribution: ""
 
 ################################################################################
 # Optional: Automatic detection of additional metric sources.
+# We collect k8s control plane metrics by default, you can disable this by
+# setting autodetect.k8sControlPlane=false. This k8sControlPlane integration
+# relies on having access the k8s control plane which k8s clusters run as a
+# managed service do not support.
 # Set autodetect.prometheus=true if you want the otel-collector agent to scrape
 # prometheus metrics from pods that have prometheus-style annotations like
 # "prometheus.io/scrape".
 # Set autodetect.istio=true in istio environment.
 ################################################################################
 
 autodetect:
+  k8sControlPlane: true
```

Comment:
I would call it just `controlPlane` , it's everything about k8s in this repo :)

Reply:
Updated to controlPlane.